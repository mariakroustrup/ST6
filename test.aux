\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand*{\memsetcounter}[2]{}
\catcode`"\active
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\@writefile{toc}{\changetocdepth  {2}}
\select@language{danish}
\@writefile{toc}{\select@language{danish}}
\@writefile{lof}{\select@language{danish}}
\@writefile{lot}{\select@language{danish}}
\newlabel{sec:flowhaandtering}{{}{1}{Flowdiagram-håndtering}{section*.2}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {0.1}{\ignorespaces Illustration af de anvendte former i flowcharts.\relax }}{1}{figure.caption.3}}
\providecommand*\caption@xref[2]{\@setref\relax\@undefined{#1}}
\newlabel{fig:flow}{{0.1}{1}{Illustration af de anvendte former i flowcharts.\relax }{figure.caption.3}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {0.0.1}Tr\IeC {\r a}dl\IeC {\o }s kommunikation}{1}{subsection.0.0.1}}
\@writefile{lot}{\contentsline {table}{\numberline {0.1}{\ignorespaces Data over afstandstest for den tr\IeC {\r a}dl\IeC {\o }se kommunikation. Venstre s\IeC {\o }jle oplyser afstand mellem mikrokontroller og modtagerenhed. Den h\IeC {\o }jre oplyser, hvorvidt transmissionen har v\IeC {\ae }ret succsesfuld eller ej.\relax }}{2}{table.caption.4}}
\newlabel{tab:traadloes_komm_test_afstand}{{0.1}{2}{Data over afstandstest for den trådløse kommunikation. Venstre søjle oplyser afstand mellem mikrokontroller og modtagerenhed. Den højre oplyser, hvorvidt transmissionen har været succsesfuld eller ej.\relax }{table.caption.4}{}}
\@writefile{lox}{\contentsline {fixme}{Note: skal denne forsinkelse v\IeC {\ae }re st\IeC {\o }rre eller mindre?? - HUSK at \IeC {\ae }ndre i br\IeC {\o }dteksten ogs\IeC {\r a}!}{2}{table.caption.4}}
\bibdata{kilder}
\newlabel{LastPage}{{}{2}{}{page.2}{}}
\xdef\lastpage@lastpage{2}
\xdef\lastpage@lastpageHy{2}
\memsetcounter{lastsheet}{2}
\memsetcounter{lastpage}{2}
