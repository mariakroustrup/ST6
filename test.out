\BOOKMARK [0][-]{indhold.0}{Indholdsfortegnelse}{}% 1
\BOOKMARK [0][-]{chapter.1}{Bilag}{}% 2
\BOOKMARK [1][-]{section.1.1}{Lungernes anatomi og fysiologi}{chapter.1}% 3
\BOOKMARK [2][-]{subsection.1.1.1}{Ventilation}{section.1.1}% 4
\BOOKMARK [2][-]{subsection.1.1.2}{Respiration}{section.1.1}% 5
