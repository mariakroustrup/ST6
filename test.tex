%implementing document formatting:
\input{preambleNy.tex}
\input{macros.tex}
\raggedbottom % Soerger for at LaTeX ikke "straekker" teksten

\begin{document}

\frontmatter	 % Forindhold - nummereres med romertal
\input{rapportAfsnit/xFormaliteter/Forside}
\clearpage

%%%% Indholdsfortegnelse (TOC) %%%%
\phantomsection													% Kunstigt afsnit, som hyperlinks kan 'holde fast i'
\pdfbookmark[0]{Indholdsfortegnelse}{indhold}					% Tildeler en klikbar bookmark til den endelige PDF
\tableofcontents*												% Indholdsfortegnelsen (kaldet To) 
%\clearpage
%\addtocontents{toc}{\protect\newpage}							% Fremtvinger sideskift i ToC hvis noedvendig (der hvor koden placeres)

\mainmatter

%Introduktion--------------------------------


%-----------------------Problemanalyse---------------------------


%%-----------------------System udvikling-------------------------



%%-----------------------Teori-------------------------




%%-----------------------Implementering-------------------------


%%-----------------------Test-------------------------%%
\input{rapportAfsnit/cProblemanalyse/Lungerne}

%\iffalse
\begingroup
\raggedright

\bibliographystyle{unsrtnat}
%\bibliography{kilder}

\endgroup

%-----------------------Bilag-------------------------
\appendix
%\chapter{Bilag}


%\fi
\end{document}
