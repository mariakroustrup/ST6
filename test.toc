\changetocdepth {2}
\select@language {danish}
\contentsline {chapter}{\chapternumberline {1}Bilag}{1}{chapter.1}
\contentsline {section}{\numberline {1.1}Lungernes anatomi og fysiologi}{1}{section.1.1}
\contentsline {subsection}{\numberline {1.1.1}Ventilation}{1}{subsection.1.1.1}
\contentsline {subsubsection}{Inspiration}{1}{section*.3}
\contentsline {subsubsection}{Eksspiration}{1}{section*.4}
\contentsline {subsection}{\numberline {1.1.2}Respiration}{1}{subsection.1.1.2}
\contentsline {subsubsection}{Transport af ilt}{2}{section*.5}
