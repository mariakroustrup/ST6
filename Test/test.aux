\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand*{\memsetcounter}[2]{}
\catcode `"\active 
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\@writefile{toc}{\changetocdepth  {2}}
\select@language{danish}
\@writefile{toc}{\select@language{danish}}
\@writefile{lof}{\select@language{danish}}
\@writefile{lot}{\select@language{danish}}
\bibstyle{unsrtnat}
\@writefile{lof}{\addvspace {10pt}}
\@writefile{lot}{\addvspace {10pt}}
\@writefile{toc}{\contentsline {chapter}{\chapternumberline {1}Lungernes anatomi og fysiologi}{1}{chapter.1}}
\@writefile{toc}{\contentsline {section}{\numberline {1.1}Lungernes opbygning}{1}{section.1.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1.1}Ventilation}{1}{subsection.1.1.1}}
\@writefile{toc}{\contentsline {subsubsection}{Inspiration}{1}{section*.3}}
\@writefile{toc}{\contentsline {subsubsection}{Eksspiration}{1}{section*.4}}
\newlabel{LastPage}{{}{1}{}{page.1}{}}
\xdef\lastpage@lastpage{1}
\xdef\lastpage@lastpageHy{1}
\memsetcounter{lastsheet}{3}
\memsetcounter{lastpage}{1}
