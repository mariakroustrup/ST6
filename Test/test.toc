\changetocdepth {2}
\select@language {danish}
\contentsline {chapter}{\chapternumberline {1}Lungernes anatomi og fysiologi}{1}{chapter.1}
\contentsline {section}{\numberline {1.1}Lungernes opbygning}{1}{section.1.1}
\contentsline {subsection}{\numberline {1.1.1}Ventilation}{1}{subsection.1.1.1}
\contentsline {subsubsection}{Inspiration}{1}{section*.3}
\contentsline {subsubsection}{Eksspiration}{1}{section*.4}
