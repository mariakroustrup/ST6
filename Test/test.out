\BOOKMARK [0][-]{indhold.0}{Indholdsfortegnelse}{}% 1
\BOOKMARK [0][-]{chapter.1}{Lungernes anatomi og fysiologi}{}% 2
\BOOKMARK [1][-]{section.1.1}{Lungernes opbygning}{chapter.1}% 3
\BOOKMARK [2][-]{subsection.1.1.1}{Ventilation}{section.1.1}% 4
